src/app/viam-client.test.ts

```diff
@@ -29,7 +29,7 @@ describe('ViamClient', () => {
     payload: 'testAccessToken',
   };
 
-  const subject = () => createViamClient(options!);
+  const subject = async () => createViamClient(options!); // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
This is probably better as an override for `*.test.ts` files in the eslint configuration rather than a one-off disable comment.

Alternatively, if the project is reorganized a little to follow our test file conventions from other FE projects, the base config will automatically disable this rule in all tests.

- `src`
  - `some-dir`
    - `some-module.ts`
    - `__tests__`
      - `some-module.spec.ts`